package.json

```diff
@@ -14,7 +14,7 @@
   },
   "scripts": {
     "postinstall": "node --preserve-symlinks --preserve-symlinks-main tools/bazel/postinstall-patches.js",
-    "build": "gulp build-release-packages",
+    "build": "bash ./scripts/build-packages-dist.sh",
```

Comment:
I could see arguments that we should not use a bash script here, but the reality is that Bash is required anyway. Bazel requires Bash to be available in the `env`.

It would make sense to make this a NodeJS script though.. but I'd say that's not a priority right now.